GettingStartedGuide.md

```diff
@@ -71,14 +73,18 @@ After you have followed the instructions in the AWS IoT Core Setup Guide, you wi
 ### 2.2 Configure the project with the AWS IoT Thing Name and AWS device Endpoint
 
 The demo will connect to the AWS IoT device Endpoint that you configure here.
+> Note: remember to set your build target using `idf.py set-target <target>`
```

Comment:
If this is being added to tell users how to use a board other than the C3 we should include a comment about running `list-target` as well

Reply:
at the top read me we are claiming that this will work  and has been tested on on ESP32, C3 and S3